### DISCUSSION TITLE
File Upload Size Increase

### DISCUSSION THREAD

**Discussion by abbas-khan252** [snipe/snipe-it#13083] [General]
How can i increase the maz file size in snipeit. it is showing 2M. How can i increase this from 2M to 10M.

**Comment by marcusmoore** [comment]
This limit is probably happening due to PHP's configuration. Check out this [StackOverflow answer](https://stackoverflow.com/a/2184541) for some details on how to change it.

**Comment by abbas-khan252** [comment]
I have tried this but the size in not increasing. this application is on
php 7.4. is this the reason that it is not working?

**Reply by marcusmoore** [reply]
Are you using apache or nginx?

**Comment by abbas-khan252** [comment]
apache

**Reply by marcusmoore** [reply]
I'm not too familiar with apache so I don't have an answer off the top of my head. In addition to `upload_max_filesize` and `post_max_size` in PHP's .ini did you change the values mentioned [here](https://stackoverflow.com/a/2185500) in `.htaccess`?

**Comment by Malvineous** [comment]
If you're using the Docker container you can add `PHP_UPLOAD_LIMIT=64` to the end of `.env` to set the limit to 64 MB.

(just adding this here since it's one of the top results for that search)
